doc/tutorials/cloud/01_import_data.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p>You can probably use JupySQL %%sql for running the above query.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>